service/src/main/java/uk/gov/hmcts/reform/fpl/model/payment/fee/FeeParameters.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.model.payment.fee;
```

Comment:
Small comment, I don't like it leaving in fpl.model it's not really FPL model is it?